demo/app/Sharp/Dashboard/Commands/ExportToCsvCommand.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Sharp\Dashboard\Commands;
```

Comment:
No to the whole command: there is a better one (with stream download) in PR https://github.com/code16/sharp/pull/399